forecasttools/trajectories_to_quantiles.py

```diff
@@ -11,47 +16,77 @@ def trajectories_to_quantiles(
     quantile_level_name: str = "quantile_level",
     id_cols: list[str] = None,
 ) -> pl.DataFrame:
+    """
+    Converts the value columns of a polars
+    dataframe of epiweekly draws (trajectories)
+    to quantile values over specified quantile
+    levels.
+
+    trajectories
+        A polars dataframe of epiweekly draws for
+        a single jurisdiction.
+    quantiles
+        A list of quantiles for the output values.
+        Defaults to None.
```

Comment:
Is this accurate? When it is none, it gets replaced with some other default values.